test/test-utils.ts

```diff
@@ -221,7 +223,9 @@ export class TestUtils {
     authProcessId: string,
     hcpId: string,
     storage?: StorageFacade<string>,
-    keyStorage?: KeyStorageFacade
+    keyStorage?: KeyStorageFacade,
+    recaptcha?: string,
+    recaptchaType?: RecaptchaType
```

Comment:
Put default values as: 
```
recaptcha?: String = 'a58afe0e-02dc-431b-8155-0351140099e4',
recaptchaType?: RecaptchaType = 'recaptcha'
```
to make sure the default use case is picked when creating users

Reply:
done